package.json

```diff
@@ -20,12 +20,14 @@
     "@types/vfile-message": "^2.0.0",
     "antd": "^3.26.0",
     "classnames": "^2.2.6",
-    "react": ">=16.0.0",
     "react-intl": "^3.9.2",
     "react-spring": "^8.0.27",
     "react-use-gesture": "^7.0.15",
     "uuid": "^3.3.3"
   },
+  "peerDependencies": {
+    "react": ">=16.0.0"
+  },
   "devDependencies": {
     "@types/classnames": "^2.2.9",
     "@types/uuid": "^3.4.6",
```

Comment:
add related packages in devDependencies otherwise will break the dev env.

```
    "react": "^16.5.2",
    "react-dom": "^16.5.2"
```

Reply:
> add related packages in devDependencies otherwise will break the dev env.
> 
> ```
>     "react": "^16.5.2",
>     "react-dom": "^16.5.2"
> ```

Care to enlighten me on why we are using different versions for the dev environment?

Reply:
We use  ```>=16.0.0``` in peerDependencies for as compatible as possible with other projects and the version in devDependencies just a more specific version to prevent frequently changing in dev process.